mobile/src/stateMachine/views/strategies/ReviewStrategiesView.tsx

```diff
@@ -97,7 +110,14 @@ export class ReviewStrategiesView extends ViewState {
       <MasloPage style={this.baseStyles.page} onBack={this.onBack} theme={this.theme}>
         <Container style={[{ height: this._contentHeight, paddingTop: 10 }]}>
           {this.renderInnerContent('Here are your focus strategies.', true)}
-          {this.showButton && <Button title='CONTINUE' style={styles.selectButton} onPress={() => this.onNext()} theme={this.theme} />}
+          {this.showButton && <Button title='CONTINUE' style={styles.selectButton} disabled={this.buttonsDisabled} onPress={() => this.onNext()} theme={this.theme} />}
+          {this.buttonsDisabled &&
+            <Portal>
+              <View style={{ width: '100%', height: '100%', flexDirection: 'column', justifyContent: 'center', alignItems: 'center' }}>
+                <Text style={[TextStyles.labelMedium, { padding: 20, backgroundColor: 'white', borderRadius: 10, borderWidth: 1, borderColor: 'white' }]}>Loading...</Text>
+              </View>
+            </Portal>
+          }
```

Comment:
Just making sure... this is not related to user events right?

Reply:
not directly related but still an important change